numba_dppy/examples/atomic_op.py

```diff
@@ -13,25 +13,34 @@
 # limitations under the License.
 
 import numpy as np
-
-import numba
-import numba_dppy, numba_dppy as dppy
-import unittest
+import numba_dppy as dppy
 import dpctl
 
 
 def main():
+    """
+    The example demonstrates the use of numba_dppy's ``atomic_add`` intrinsic
+    function on a SYCL GPU device. The ``dpctl.select_gpu_device`` is
+    equivalent to ``sycl::gpu_selector`` and returns a sycl::device of type GPU.
+    """
+
     @dppy.kernel
     def atomic_add(a):
         dppy.atomic.add(a, 0, 1)
 
     global_size = 100
     a = np.array([0])
 
-    with dpctl.device_context("opencl:gpu") as gpu_queue:
-        atomic_add[global_size, dppy.DEFAULT_LOCAL_SIZE](a)
-        # Expected 100, because global_size = 100
-        print(a)
+    try:
+        d = dpctl.select_gpu_device()
+        with dpctl.device_context(d):
+            print("Offloading to ...")
+            d.print_device_info()
+            atomic_add[global_size, dppy.DEFAULT_LOCAL_SIZE](a)
+            # Expected 100, because global_size = 100
+            print(a)
+    except ValueError:
+        print("No OpenCL GPU found.")
```

Comment:
How do we know the device is of OpenCL type in this case?

Reply:
done.